### ISSUE TITLE
More Filter Icons

### ISSUE THREAD

**Issue by ig-robstoffers** [IgniteUI/material-icons-extended#49]
There are more icons that I think we should add that pertain to different filter operations.

### Date Related Icons
this-quarter (filter showing dates for the current quarter)
next-quarter
last-quarter
this-week (filter showing dates for the current week)
next-week
last-week
year-to-date (filter showing dates from beginning of year to current date)
quarter-one (filter showing dates in Q1)
quarter-two
quarter-three
quarter-four
month (filter for a specific month, user provides the desired month)
year (filter for a specific year, user provides the desired year)

### General Icons
top (filter showing the top X records, where X is a user provided number)
top-percentile (filter showing the top X% records, where X is a percent... if that makes sense)
bottom
bottom-percentile

**Comment by StefanIvanov** [comment]
These should go into the editor category.

**Comment by ig-robstoffers** [comment]
@sdimchevski I wanted to confirm whether it was understood what the `last-quarter` and `last-week` icons meant.  When I requested these I meant them to really mean the previous quarter and previous week.  Not the very last quarter of the year or last week of the month.  From what I can see in the new icons it seems that we thought it meant "last" instead of "previous".

I didn't explain what I intended them to represent in the beginning so I apologize if there was any confusion.  Could we update those icons, or add new ones for `previous-quarter` and `previous-week`?
